product/models.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+from user.models import User
+
+class MainMenu(models.Model): 
+    email = models.CharField(max_length = 200, help_text = "이메일")
+    class Meta: 
+        db_table = "main_menus"
```

Comment:
```suggestion
class MainMenu(models.Model): 
    email = models.CharField(max_length = 200, help_text = "이메일")

    class Meta: 
    db_table = "main_menus"
```

내부 클래스 선언 전에 한 줄 띄워주세요. 아래 다른 클래스들도 마찬가지입니다 :)